src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -390,15 +394,16 @@ UCS_PROFILE_FUNC_ALWAYS(ucs_status_t, uct_ib_mlx5_devx_reg_atomic_key,
     if (memh->super.flags & UCT_IB_MEM_MULTITHREADED) {
         return uct_ib_mlx5_devx_reg_ksm_data(md, is_atomic, memh->address,
                                              mr->ksm_data, mr->ksm_data->length,
-                                             uct_ib_md_atomic_offset(mr_id),
+                                             atomic_offset,
```

Comment:
change uct_ib_mlx5_devx_reg_ksm_data to also get iova instead of offset - for consistency
and change uct_ib_mlx5_devx_reg_ksm argument from `uintptr_t address` to `uint64_t iova`